.gitignore

```diff
@@ -14,6 +14,7 @@ __pycache__/
 htmlcov/
 .tox/
 .nox/
+.pyc
```

Comment:
Why is this necessary? `__pycache__` is already included,  what produces `.pyc` files outside `__pycache__`?

Reply:
It is not necessary, I will remove it.